compatibilityTests/src/test/scala/org/apache/spark/sql/SnappyWholeStageCodegenSuite.scala

```diff
@@ -20,11 +20,4 @@ import org.apache.spark.sql.execution.WholeStageCodegenSuite
 import org.apache.spark.sql.test.{SharedSnappySessionContext, SnappySparkTestUtil}
 
 class SnappyWholeStageCodegenSuite extends WholeStageCodegenSuite
-    with SharedSnappySessionContext with SnappySparkTestUtil {
-
-  override def ignored: Seq[String] = Seq(
-    "Aggregate should be included in WholeStageCodegen",
```

Comment:
Can you please explain briefly why they used to fail before and why do they pass now?

Reply:
Primarily spark-side changes to update the test that searches for WholeStageCodegenExec rather than assuming top-level to be it. Reason is that we have a top-level wrapper plan "CodegenSparkFallback"